x-pack/plugins/event_log/server/plugin.ts

```diff
@@ -91,8 +109,26 @@ export class Plugin implements CorePlugin<IEventLogService> {
       event: { action: ACTIONS.starting },
       message: 'eventLog starting',
     });
+
+    this.eventLogClientService = new EventLogClientService({
+      esContext: this.esContext,
+      savedObjectsService: core.savedObjects,
+    });
+    return this.eventLogClientService;
```

Comment:
I wonder if we should expose explicit functions of the `eventLogClientService` like `getEventLogClient` instead to allow future variables to also be part of the start contract.

Reply:
Yeah, I thought of that, but chose to align with existing Setup api which exposes a service.
I'll rename `eventLogClientService` to `eventLogStartService`

Reply:
That makes sense now

Reply:
We may want to change the shape of the setup/start interfaces to make them more aligned - I guess changing the setup to return a `eventLogSetupService` or something.  We'll see ...